app/src/main/res/values-de/strings.xml

```diff
@@ -158,11 +158,11 @@
     <string name="you_re_apparantly_a_developer">Sie sind anscheinend ein Entwickler</string>
     <string name="developer_settings">Entwicklereinstellungen</string>
     <string name="migration">Migration</string>
-    <string name="set_a_password_for_your_daily_backup">Legen Sie ein Passwort für Ihre tägliche Sicherung fest</string>
-    <string name="set_a_password_used_to_encrypt_your_daily_backup_that_is_written_to_external_storage">Legen Sie ein Passwort fest, das zur Verschlüsselung Ihrer täglichen Sicherung verwendet wird, die auf dem externen Speicher geschrieben ist.</string>
-    <string name="backup_password">Sicherungspasswort</string>
-    <string name="restore_from_automatic_backup">Von automatischer Sicherung wiederherstellen</string>
-    <string name="it_appears_an_automatic_backup_exists_on_your_device_if_you_would_like_to_restore_enter_your_backup_password">Es scheint, dass eine automatische Sicherung auf Ihrem Gerät vorhanden ist. Wenn Sie diese wiederherstellen möchten, geben Sie Ihr Sicherungspasswort ein.</string>
+    <string name="set_a_password_for_your_daily_backup">Legen Sie ein Passwort für Ihr tägliches Backup fest</string>
+    <string name="set_a_password_used_to_encrypt_your_daily_backup_that_is_written_to_external_storage">Legen Sie ein Passwort fest, das zur Verschlüsselung Ihres täglichem Backup verwendet wird, die auf dem externen Speicher geschrieben ist.</string>
+    <string name="backup_password">Backuppasswort</string>
```

Comment:
I don't agree with changes to this part of the text since is is more clear to a broader audience in the way it was before. But I agree with the it needing to be changed.
I would suggest just fixing the current grammar to Sicherungskopie since it is the actual translation to backup.

Additional Resources: https://www.duden.de/rechtschreibung/Sicherungskopie

And the correct spelling would be Backup-Passwort.